src/dune_pkg/fetch.ml

```diff
@@ -0,0 +1,73 @@
+open Stdune
+module Process = Dune_engine.Process
+module Display = Dune_engine.Display
+
+module FiberJob = struct
+  let run command =
+    let open Fiber.O in
+    let prog =
+      command |> OpamProcess.command_cmd |> Path.Outside_build_dir.of_string
+      |> Path.outside_build_dir
+    in
+    let args = OpamProcess.command_args command in
+    let dir =
+      command |> OpamProcess.command_dir
+      |> Option.map ~f:(fun path ->
+             path |> Path.Outside_build_dir.of_string |> Path.outside_build_dir)
+    in
+    let display = Display.Quiet in
+    let env = command |> OpamProcess.command_env |> Option.map ~f:Env.of_unix in
+    let stdin_from =
+      match command |> OpamProcess.command_allow_stdin with
+      | Some true -> Some Process.Io.stdin
+      | None | Some false -> None
+    in
+    let stderr_to = Process.Io.stderr in
+    let stdout_to = Process.Io.stdout in
```

Comment:
Is that dune's stdout/stderr? If so, that's definitely not allowed as it will ruin the display.

Reply:
This was just temporary to get something working. I've fixed it since.